forbidigo/forbidigo.go

```diff
@@ -97,19 +99,20 @@ type visitor struct {
 	linter   *Linter
 	comments []*ast.CommentGroup
 
+	pass   *analysis.Pass
 	fset   *token.FileSet
 	issues []Issue
 }
 
-func (l *Linter) Run(fset *token.FileSet, nodes ...ast.Node) ([]Issue, error) {
-	var issues []Issue 
+func (l *Linter) Run(pass *analysis.Pass, nodes ...ast.Node) ([]Issue, error) {
```

Comment:
Just wanted to note that this is a breaking change for the linter so we'll have to bump the major version.  This also embeds the concept of an analysis pass into the code that wasn't previously there.  My preference would be to pass in the `TypesInfo` directly so that we aren't dependent on how analysis code is organized.  It has been the position of the code so far that the linter is not specific to the golang analyzer package.

Reply:
Then https://github.com/ashanbrown/forbidigo/pull/22 will have to add `*types.Package`. I agree that it would be consistent with how the code is currently organized, but perhaps it would become simpler if it was more integrated with the analyzer package?

Just a thought, I'll add separate parameters in both PRs for now.